running.html

```diff
@@ -166,15 +166,14 @@
                 <li>Add the following line to <tt>/etc/apt/sources.list</tt>. Replace the distribution name as appropriate.
                   <pre>deb http://repo-cockpitproject.rhcloud.com/debian/ jessie main</pre></li>
                 <li>Import Cockpit's signing key to the apt sources keyring:
-                  <pre>sudo apt-key adv --keyserver sks-keyservers.net --recv-keys F1BAA57C</pre></li>
+                  <pre>sudo apt-key adv --keyserver sks-keyservers.net --recv-keys 0D2A45C3F1BAA57C</pre></li>
+                <li>Verify fingerprint:
+                   <pre>sudo apt-key finger F1BAA57C</pre>
+                   <p>Compare the output: <span style="font-family: monospace">Key fingerprint = FD9A 5764 17F7 B1D8 63C4  7A5A 0D2A 45C3 F1BA A57C</span></p></li>
```

Comment:
I recommend using the `<tt>` tag, like [above](https://github.com/cockpit-project/cockpit-project.github.io/pull/64/files#diff-7dae741232cc9134b09ef23e3e660e5eR166)